src/styles/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+  background-color: var(--card-background-color);
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.card__product-title {
+  font-size: 12px;
+  color: var(--text-color);
+  margin-bottom: 4px;
+  line-height: 18px;
+  flex-grow: 1;
```

Comment:
this line has been brought unintended way from figma IMHO
...
all in all fine BEM and pretty straightforward logical structure
8-)

Reply:
All works fine w/o this line, so... fixed